.github/workflows/tests.yml

```diff
@@ -265,10 +265,10 @@ jobs:
         run: yarn workspace ${{ matrix.example }}-example start & npx wait-on -c waitOnConfig.json -t 20000 http-get://localhost:3000/ui/components/authenticator/sign-in-with-username
 
       - name: Run E2E tests against ${{ matrix.example }} example
-        run: yarn workspace e2e test:authenticator
+        run: yarn workspace e2e test:examples
```

Comment:
Updated this to run all e2e tests which run against the examples apps.

Reply:
It won't run against vue and angular, because I only added the `@react` tag to the hook tests.